### ISSUE TITLE
Where is training codes?

### ISSUE THREAD

**Issue by Winckel** [PeirongLiL/BI-RADS-Diagnosis-for-Mammograms#1]
In test_ddsm.py and test_inbreast.py, the code only includes validation set evaluation-related parts and does not contain the model training code, optimizer, loss function, or training loop. 
Could you also upload your complete training code?

**Comment by PeirongLiL** [comment]
> In test_ddsm.py and test_inbreast.py, the code only includes validation set evaluation-related parts and does not contain the model training code, optimizer, loss function, or training loop. Could you also upload your complete training code?

训练代码在旧的实验室，我下周二发您。

**Comment by PeirongLiL** [comment]
> 好的, 感谢您的帮助, 你们这篇BIRADS分级的文献具有很大的学术参考价值和应用意义.
> 
> 
> 
> 在 2025-06-20 16:58:33，"fjnu_mia" ***@***.***> 写道：
> 
> PeirongLiL left a comment ([PeirongLiL/BI-RADS-Diagnosis-for-Mammograms#1](https://github.com/PeirongLiL/BI-RADS-Diagnosis-for-Mammograms/issues/1))
> 
> In test_ddsm.py and test_inbreast.py, the code only includes validation set evaluation-related parts and does not contain the model training code, optimizer, loss function, or training loop. Could you also upload your complete training code?
> 
> 训练代码在旧的实验室，我下周二发您。
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>

我已经将具体的训练代码（train.py）添加在仓库中
